src/routes/index.tsx

```
@@ -21,6 +21,7 @@
 import userAccessRoutes from './page-routes/user-access';
 import { RouteErrorBoundry } from './RouteErrorBoundary';
 import { GithubRedirectRouteParams } from './utils';
+import issuesRoutes from './page-routes/issues';
```

ESLint

[error] `./page-routes/issues` import should occur before import of `./page-routes/namespace`

(import/order)